app/models/room.rb

```diff
@@ -28,4 +28,8 @@ def full_name
     [floor.building.nick_name, room_number].join(' ')
   end
 
+  def room_state?
+    RoomState.find_by(room_id: self).present?
```

Comment:
self or `self.id` ?

Reply:
@srmoola Since the Room model has a `belongs_to :room` statement we can use the room object here. Even this will work:  `RoomState.find_by(room: self).present?`
But I don't know if some methods are better than others.
You can try in bin/rails console:
```
room = Room.find(RoomState.last.room_id)
RoomState.find_by(room: room)
```